Add missing integration test aliases files

felixfontein opened the pull request:
##### SUMMARY
Ref: https://github.com/ansible-collections/community.general/pull/11346#discussion_r2652823976

`callback` doesn't need one since it's a helper, similar to the `setup_*` roles.

##### ISSUE TYPE
- Test Pull Request

##### COMPONENT NAME
integration tests

russoz commented:
I am missing `tests/integration/targets/filter_json_patch/aliases`, but when I added it to that other PR it popped up errors.

felixfontein commented:
> I am missing `tests/integration/targets/filter_json_patch/aliases`, but when I added it to that other PR it popped up errors.

Hmm, you didn't mention that one in the comment; I only considered the list there... :) I'll add it later.

> why have does .pem files been moved?

It's nost just the `.pem` files that were moved. I renamed the whole directory, it was incorrectly named. The `lookup_` prefix was missing.

felixfontein commented:
@russoz thanks for noticing this and reviewing!
